CODE_OF_CONDUCT.md

```diff
@@ -1,74 +1,86 @@
-# Contributor Covenant Code of Conduct
+# コントリビューター行動規範
```

Comment:
Should the CoC be written in Japanese?

Reply:
Good question.
I think it should written in Japanese and English both.